src/controllers/RideOffer.js

```diff
@@ -1,52 +1,125 @@
 import Model from '../models/model';
 
-const driverOfferModel = new Model('ride_offer');
+const offerModel = new Model('ride_offer');
+/**
+ * @description - This method add offer to the database
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @return {object} - it return data object
+ */
 export const addOffer = async (req, res) => {
-  const {
-    driverId, destination, price
-  } = req.body;
-  const columns = 'driver_id, destination, price';
-  const values = `'${driverId}', '${destination}', '${price}'`;
+  const { data: { id } } = req.user;
+  const { destination, price } = req.body;
+  const columns = '"driverId", destination, price';
+  const values = `'${id}', '${destination}', '${price}'`;
   try {
-    const data = await driverOfferModel.insertWithReturn(columns, values);
-    res.status(200).json({ msg: data.rows });
+    const data = await offerModel.insertWithReturn(columns, values);
+    return res.status(201)
+      .json({ data: data.rows[0], message: 'offer created successfully', success: true });
   } catch (err) {
-    res.status(400).json({ msg: err.stack });
+    res.status(500).json({ message: err.message });
   }
 };
-
+/**
+ * @description - This method will return a offer for particular driver
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @return {object} - it return object
+ */
 export const DriverRideOfferPage = async (req, res) => {
```

Comment:
This function seems off compared to others, do you mean to use `getDriverOffers`?

Reply:
yes